etc/Makefile.am

```diff
@@ -9,5 +9,8 @@
 
 MAINTAINERCLEANFILES    = Makefile.in
 
+configdir		= @CONFIGDIR@
+dist_config_DATA	= sysconfig/crm_mon sysconfig/pacemaker
```

Comment:
I figured this was a cleaner way of installing the file, but I suppose one downside is that if someone is running "make install" from new Pacemaker source on an existing cluster node, it will overwrite their existing sysconfig file(s). This isn't an issue for distribution packagers or someone building their own packages on a non-cluster-node host, only for someone installing from source directly on a node. I'm not sure if that's something to be concerned about or not.

Reply:
It would lose the cleaner look, but I could install these via install-exec-local instead, and only install if not already present. Basically mimic rpm's noreplace.

Reply:
If I were managing a serious HA setup somewhere, I would (1) install from my distribution's paid support packages and (2) install the next version to a test cluster first to make sure nothing went wrong.  If I were managing a less serious cluster, I think I'd still be installing from distributed packages.  If I were more of a hobbyist, I might be installing from source, but I'm not sure I would build directly on a cluster node.  So I'm not sure there's anything really worth worrying about here.  If you're really concerned maybe send off an email to the list and see if anyone's doing this.

Reply:
Intellectually I agree, but it keeps nagging at me that it would be unexpected behavior, and someone somewhere would be cursing me. :) I'll probably go the install-exec-local route. The install command has an option for the reverse behavior, copy the existing file to .old and install the new one, too bad it doesn't have one for this. But it's easy enough to do -e $f || install.